src/middleware/initiate.js

```diff
@@ -30,14 +128,38 @@ exports.initiateContextMiddleware = () => async (ctx, next) => {
     return
   }
 
+  const endpointState = await readStateByEndpointId(endpoint._id)
+
+  logger.info(`${endpoint.name} (${requestUUID}): Initiating new request`)
```

Comment:
@BMartinos This log can't use the requestUUID and needs to come after the openhim header logic below. Below, on lines 147 - 149 in this file, the `ctx.state.uuid` is set. If the request wasn't sent through the OpenHIM the header is empty and `requestUUID` is used. But going through the OpenHIM sets this valid - therefore this log has a different UUID to all the other UUIDs

```
[2020-04-17T07:13:16:685Z] INFO : mCSD Orchestration (7936f688-f457-4aad-827e-4b79b572759f): Initiating new request
[2020-04-17T07:13:16:685Z] INFO : mCSD Orchestration (5e99570cc0d2ef1dfb244122): Parsing incoming body into JSON format for processing
[2020-04-17T07:13:16:789Z] INFO : mCSD Orchestration (5e99570cc0d2ef1dfb244122): Successfully performed request/s
```

Reply:
aaa well spotted. Will address now :+1: